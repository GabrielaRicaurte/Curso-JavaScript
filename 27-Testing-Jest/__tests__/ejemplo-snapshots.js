const cliente = {
    
    nombre: 'Gaby Ruiz',
    balance: 500,
    tipo: 'Premium'

};

describe('Testing al Cliente', ()=> {
    test('Es Gaby', ()=> {
        expect(cliente).toMatchSnapshot();
    })
}); 


/* 
    Para actulizar los datos guardados en el snapshot se escribe en la terminal npm t -- -u
*/