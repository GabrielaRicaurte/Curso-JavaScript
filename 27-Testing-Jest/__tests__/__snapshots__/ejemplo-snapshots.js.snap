// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al Cliente Es Gaby 1`] = `
{
  "balance": 500,
  "nombre": "Gaby Ruiz",
  "tipo": "Premium",
}
`;
